docs/team/lysire.md

```diff
@@ -3,44 +3,62 @@ layout: page
+    not only required changes to existing commands but changes to support handling of keyboard shortcuts.
 
-* **Code contributed**: [RepoSense link]()
+* **Code contributed**: [RepoSense link](https://nus-cs2103-ay2021s1.github.io/tp-dashboard/#search=lysire&sort=groupTitle&sortWithin=title&since=2020-08-14&timeframe=commit&mergegroup=&groupSelect=groupByRepos&breakdown=false&tabOpen=true&tabType=zoom&zA=Lysire&zR=AY2021S1-CS2103-T16-3%2Ftp%5Bmaster%5D&zACS=370.3076923076923&zS=2020-08-14&zFS=lysire&zU=2020-11-06&zMG=false&zFTF=commit&zFGS=groupByRepos&zFR=false)
 
 * **Project management**:
-  * Managed releases `v1.3` - `v1.5rc` (3 releases) on GitHub
-
-* **Enhancements to existing features**:
-  * Updated the GUI color scheme (Pull requests [\#33](), [\#34]())
-  * Wrote additional tests for existing features to increase coverage from 88% to 92% (Pull requests [\#36](), [\#38]())
+  * Contributed to releases `v1.2` - `v1.4` (3 releases) on GitHub
 
 * **Documentation**:
   * User Guide:
-    * Added documentation for the features `delete` and `find` [\#72]()
-    * Did cosmetic tweaks to existing documentation of features `clear`, `exit`: [\#74]()
+    * Added documentation for the features `undo`, `redo` and `history` [\#119](https://github.com/AY2021S1-CS2103-T16-3/tp/pull/119)
+    * Did cosmetic tweaks to existing documentation for the `rooms` command e.g. [\#91](https://github.com/AY2021S1-CS2103-T16-3/tp/pull/91)
   * Developer Guide:
-    * Added implementation details of the `delete` feature.
+    * Added implementation details of the `undo`/`redo` feature. [\#119](https://github.com/AY2021S1-CS2103-T16-3/tp/pull/119)
+    * Added class diagram that shows the class structure regarding the `undo`/`redo` feature. [\#181](https://github.com/AY2021S1-CS2103-T16-3/tp/pull/181)
+    * Updated storage class diagram. [\#132](https://github.com/AY2021S1-CS2103-T16-3/tp/pull/132)
 
 * **Community**:
-  * PRs reviewed (with non-trivial review comments): [\#12](), [\#32](), [\#19](), [\#42]()
-  * Contributed to forum discussions (examples: [1](), [2](), [3](), [4]())
-  * Reported bugs and suggestions for other teams in the class (examples: [1](), [2](), [3]())
-  * Some parts of the history feature I added was adopted by several other class mates ([1](), [2]())
-
-* **Tools**:
-  * Integrated a third party library (Natty) to the project ([\#42]())
-  * Integrated a new Github plugin (CircleCI) to the team repo
+  * PRs reviewed (with non-trivial review comments): [\#178](https://github.com/AY2021S1-CS2103-T16-3/tp/pull/178), 
+  [\#179](https://github.com/AY2021S1-CS2103-T16-3/tp/pull/179), 
+  [\#185](https://github.com/AY2021S1-CS2103-T16-3/tp/pull/185)
+  * Reported bugs and suggestions for other teams in the class (examples: [1](https://github.com/Lysire/ped/issues)
```

Comment:
Minor nitpick, but the closing bracket at the end of this line is missing.  I'll let you decide whether you'd rather fix this now or in the final version.

Otherwise, LGTM!

Reply:
Thanks for pointing this out! I've fixed this issue, help me merge it in please.